CMakeLists.txt

```diff
@@ -358,8 +363,23 @@ else()
   list(PREPEND VIAMCPPSDK_GRPCXX_REFLECTION_LIBRARIES ${VIAMCPPSDK_GRPCXX_REFLECTION_LIB})
 endif()
 
-find_package(xtl ${VIAMCPPSDK_XTL_VERSION_MINIMUM} CONFIG REQUIRED)
-find_package(xtensor ${VIAMCPPSDK_XTENSOR_VERSION_MINIMUM} CONFIG REQUIRED)
+include(FetchContent)
+
+FetchContent_Declare(
```

Comment:
By adding the `FIND_PACKAGE_ARGS` this should first look for system xtl/xtensor and use that if available, otherwise we will download, build, and install xtl/xtensor into the installation prefix. This seemed like the best way to go about it. Note that xtl and xtensor are header only, so this is mostly just about downloading and isntalling the headers.